transformer_engine/pytorch/fp8.py

```diff
@@ -477,6 +483,41 @@ def restore_fp8_meta_tensors(fp8_meta: Dict[str, Any]) -> None:
         fp8_meta["scaling_fwd"].scale_inv = fp8_meta["updated_scale_inv_fwd"]
 
 
+@contextmanager
+def fp8_init(enabled: bool = False) -> None:
```

Comment:
Could we rename it to something like `fp8_model_init`, to differentiate from the other `fp8_init` that's in `base.py` file? Also, the other `fp8_init` could be more descriptive like `fp8_metadata_init` or similar.